tests/test_models.py

```diff
@@ -0,0 +1,7 @@
+# -*- coding: utf-8 -*-
+from cmsplugin_articles_ai.factories import TagFactory
+
+
+def test_tag_factory_slug():
```

Comment:
Shouldn't this be `test_tag_slug`? The fact that you're using a factory shouldn't matter.